types/material-ui/index.d.ts

```diff
@@ -8667,6 +8907,8 @@ declare module "material-ui/svg-icons" {
     export import ToggleStar = __MaterialUI.SvgIcon; // require('material-ui/svg-icons/toggle/star');
     export import ToggleStarBorder = __MaterialUI.SvgIcon; // require('material-ui/svg-icons/toggle/star-border');
     export import ToggleStarHalf = __MaterialUI.SvgIcon; // require('material-ui/svg-icons/toggle/star-half');
+// }}}
+    export import NavigationArrowDropRight = __MaterialUI.SvgIcon; // require('material-ui/svg-icons/navigation-arrow-drop-right');
```

Comment:
Because it is an exception, it is out of code generation.
I do not know the reason, but only `navigation-arrow-drop-right.js` is placed in a strange path. I would like you to see https://github.com/callemall/material-ui/tree/master/src/svg-icons. Other files are `./[category]/[icon].js` but the file is `./navigation-arrow-drop-right.js` :sweat: